packages/manager/src/features/Kubernetes/KubernetesClusterDetail/UpgradeClusterDialog.tsx

```diff
@@ -100,15 +97,12 @@ export const UpgradeKubernetesClusterToHADialog = (props: Props) => {
     >
       <HACopy />
       <Typography style={{ marginBottom: 8, marginTop: 12 }} variant="body1">
-        {flags.dcSpecificPricing
-          ? `For this region, pricing for the HA control plane is $${getDCSpecificPrice(
-              {
-                basePrice: LKE_HA_PRICE,
-                flags,
-                regionId: regionID,
-              }
-            )}`
-          : `Pricing for the HA control plane is ${displayPrice(LKE_HA_PRICE)}`}
+        {`For this region, pricing for the HA control plane is $${getDCSpecificPrice(
+          {
+            basePrice: LKE_HA_PRICE,
+            regionId: regionID,
+          }
+        )}`}
         &nbsp;per month per cluster.
```

Comment:
Optional but there is potential here for a bit more simplification

```suggestion
        For this region, pricing for the HA control plane is $
        {getDCSpecificPrice({ basePrice: LKE_HA_PRICE, regionId: regionID })}{' '}
        per month per cluster
```